apps/artemis-web/HOC/withAuth.tsx

```diff
@@ -0,0 +1,59 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { extractUser } from '../lib/helpers';
+import passport from '../lib/passport';
+import auth from '../middleware/auth';
+import { initializeApollo } from '../utils/graphql';
+import nc from 'next-connect';
+import React from 'react';
+import { useCookie } from 'next-cookie';
+import jwt from 'jsonwebtoken';
+
+const withAuth = (WrappedComponent) => {
+  const Wrapper = (props) => {
```

Comment:
what is this doing?

Reply:
Right now nothing special. We maybe add logic if we figure out how to fetch and pass the token to the component